dev/AnimatedIcon/AnimatedIcon.idl

```diff
@@ -22,13 +22,17 @@ unsealed runtimeclass AnimatedIcon : Windows.UI.Xaml.Controls.IconElement
     [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
     IconSource FallbackIconSource{ get; set; };
 
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    Boolean MirroredWhenRightToLeft{ get; set; };
```

Comment:
Do we need to update any of our usages to set this property to non default ?

Reply:
Some Icons have specific **Right to Left** versions which are not just mirrors.  Is there a good way to swap out the Animated Icon and Fallback, for those cases?

Reply:
@ranjeshj yes, we will.

@mdtauk interesting point, this would be doable by the app author, but is not currently easily supported by AnimatedIcon (or any of our icons).